ssh/tailssh/tailssh.go

```diff
@@ -256,7 +257,7 @@ ProcessAction:
 		})
 		defer t.Stop()
 	}
-	srv.handleAcceptedSSH(ctx, s, ci, lu)
+	srv.handleAcceptedSSH(ctx, s, action, ci, lu)
```

Comment:
yeah, this suggests we should have a new type before this, and then `handleAcceptedSSH` can be renamed to `run` or something on that new type.